RFC/src/RFC-0100_BaseLayer.md

```diff
@@ -67,9 +67,10 @@ The Tari Base Layer network comprises the following major pieces of software:
   Tari base layer and cryptocurrency. The base nodes validate and transmit transactions and blocks, and maintain
   consensus about the longest valid proof-of-work blockchain.
 * Mining software. Mining nodes perform proof-of-work to secure the base layer and compete to submit the
-  next valid block into the Tari blockchain. Tari is merge-mined with Monero. The Tari source provides two alternatives
-  for Tari miners:
-  * A standalone miner
+  next valid block into the Tari blockchain. Tari uses two Proof of Work (PoW) algorithms, the first is merge-mined with Monero and a second native SHA3 PoW. 
+  The Tari source provides three alternatives for Tari miners:
+  * A standalone miner for SHA3 mining
+  * A merge-mining proxy to be used with XMRig to merge mine Tari with Monero
   * A stratum-compatible pool miner.
```

Comment:
Is this even still on the cards?
@StriderDM  you might know?

Reply:
@SWvheerden As far as I know it is, though it would be two alternatives. First and third bullet would exist within the same application, just as an additional mode for supported pool software. Point 3 is also still under development, so these docs may need to change again in the near future.

Reply:
Think this can go in as-is for now though :)

Reply:
Ja, I didn't know exactly what form these would take so just added what I knew was there and left the stratum miner for the future because that would exist in some form at some point.

Reply:
We also need to mention monero-pool and nodejs-pool for merge mining as well at some point.